src/oryx/install.sh

```diff
@@ -236,16 +234,19 @@ fi
 if [[ "${PINNED_SDK_VERSION}" != "" ]]; then
     rm -f ${GIT_ORYX}/global.json
     rm -rf /usr/share/dotnet/sdk/$PINNED_SDK_VERSION
-
-    # Extract the major, minor version and the first digit of the patch version
-    MAJOR_MINOR_PATCH1_VERSION=${PINNED_SDK_VERSION%??}
-    rm -rf /usr/share/dotnet/shared/Microsoft.NETCore.App/$MAJOR_MINOR_PATCH1_VERSION
-    rm -rf /usr/share/dotnet/shared/Microsoft.AspNetCore.App/$MAJOR_MINOR_PATCH1_VERSION
-    rm -rf /usr/share/dotnet/templates/$MAJOR_MINOR_PATCH1_VERSION
+    NEW_RUNTIME_VERSIONS=$(dotnet --list-runtimes | awk '{print $2}' | sort | uniq)
+    if [ -n "${RUNTIME_VERSIONS:-}" ]; then
+        SDK_INSTALLED_RUNTIME=$(echo "$NEW_RUNTIME_VERSIONS" | grep -vxFf <(echo "$RUNTIME_VERSIONS"))
+    else
+        SDK_INSTALLED_RUNTIME=$(echo "$NEW_RUNTIME_VERSIONS")
```

Comment:
> SDK_INSTALLED_RUNTIME=$(echo "$NEW_RUNTIME_VERSIONS")

Why does the assignment require `echo`, not sure if I correctly follow the logic here

Reply:
Removed echo from here. Explained the logic in the next comment